src/implementations/vanilla/components/select/select.js

```diff
@@ -361,6 +373,10 @@ export class Select {
    */
   handleClickSelectAll(e) {
     e.preventDefault();
+    // Early returns.
```

Comment:
i don't understand this "early return" logic, how is the "select all" accessible when the select is disabled?